registry/registry.go

```diff
@@ -99,7 +101,11 @@ func doRequest(req *http.Request, jar http.CookieJar, timeout TimeoutType, secur
 			return false
 		}
 
-		hostDir := path.Join("/etc/docker/certs.d", req.URL.Host)
+		if runtime.GOOS == "windows" {
+			hostDir = filepath.Join(os.TempDir(), "/docker/certs.d", req.URL.Host)
```

Comment:
Does it make sense to define these base paths as constants, then we can use build tags instead of branching here?